kong/plugins/prometheus/handler.lua

```diff
@@ -34,6 +37,26 @@ function PrometheusHandler.log(self, conf)
     end
   end
 
+  local tags = {}
+  if conf.expose_tags.from_service and message.service and message.service.tags then
+    for i = 1, #message.service.tags do
+      tags[message.service.tags[i]] = true
+    end
```

Comment:
My lua is rusty but could these not be written using `table.insert` such as

```suggestion
    for _, tag in pairs(message.service.tags) do
      table.insert(tags, tag)
    end
```

Reply:
The idea here is: do not duplicate strings; `table.insert` will insert the string as the "table value" and the index will be `index++`;

Reply:
example of the result for your suggestion: https://www.jdoodle.com/iembed/v0/C2b

Reply:
Ah alright that is not at all how I expected `table.insert` to operate :+1: